content/docs/components/introduction.md

```diff
@@ -16,18 +16,18 @@ Before we talk more about components, let's understand how they differ from comp
 
 Components are regular Edge templates created with the purpose of reuse. Components can access additional runtime properties like `$props` and `$slots`, which are unavailable to other Edge templates.
 
-We recommend you create components inside the `components` directory of your template's root path. This helps create a visual boundary between the components and the rest of the templates used by your application.
+We recommend you create components inside the `components` directory of your template's root path. This helps create a visual boundary between the components and the rest of the templates used by your application. By default, the template root path in an AdonisJS project is usually set to the `resources/views` directory. 
 
 Let's start by creating a button component. We will store it inside the `components/button.edge` file. 
 
 ```edge
 // title: views/components/button.edge
-<button type="{{ type || 'submit' }}"> {{ text }} </button>
+<button type="{{ type || 'submit' }}">{{ text }}</button>
 ```
 
 ## Using components
 
-You must use the `@component` tag to render a component inside your templates. The component tag accepts the template path as the first parameter and `props` as the second parameter.
+You must use the `@!component` tag to render a component inside your templates. The component tag accepts the template path as the first parameter and `props` as the second parameter.
```

Comment:
It should be `@component` here, since the `!` is a way to write a self closing tag.